pyaedt/modules/MaterialLib.py

```diff
@@ -482,16 +482,20 @@ def export_materials_to_file(self, full_json_path):
 
         def find_datasets(d, out_list):
             for k, v in d.items():
-                if isinstance(v, dict):
+                if isinstance(v, dict) or isinstance(v, OrderedDict):
                     find_datasets(v, out_list)
                 else:
-                    if "pwl(" in str(v):
-                        out_list.append(v[v.find("$") : v.find(",")])
+                    a = copy.deepcopy(v)
+                    val = a
+                    if str(type(a)) == r"<type 'List'>":
```

Comment:
Why won't `isinstance(a, list)` work?  Type checking through strings seems brittle.

Reply:
because it's not a Python list. this methods (and some others) are returning C# List construct. the isinstance doesn't work on it